Cabal-syntax/src/Distribution/SPDX/LicenseId.hs

```diff
@@ -2055,6 +2212,125 @@ licenseIdList LicenseListVersion_3_16 =
     ]
     ++ bulkOfLicenses
 
+licenseIdList LicenseListVersion_3_20 =
```

Comment:
I have just copied this list from LicenseListVersion_3_16. Where can I get the new list?

Reply:
@Kleidukos might know! Checking #8135 and previous PRs might help too.

Reply:
> @Kleidukos might know! Checking #8135 and previous PRs might help too.

Yes I have checked it out but couldn't figure it out and this change was not made while following this documentation https://github.com/haskell/cabal/wiki/Updating-the-license-list

Reply:
The 'Releases' sidebar at https://github.com/spdx/license-list-data say which version is the current one, so I'd just replace 'v3.17' in the docs with it and see if it works.

Reply:
yeah I just went on the SPDX website to see the list and current version